### ISSUE TITLE
Add the ability to pass a config object instead of configPath to dockerfilelint.run()

### ISSUE THREAD

**Issue by EvgenyOrekhov** [replicatedhq/dockerfilelint#92]
I am trying to use Dockerfilelint programmatically.
`dockerfilelint.run()` rereads a config using `fs.lstatSync()` and `fs.readFileSync()` on every call.
It would be more performant (and in my case more convenient) to be able to pass a config object instead of `configPath` to `dockerfilelint.run()`:

```js
// I already have `configObject` and `files`
const warnings = files.map(
    // Here `dockerfilelint.run()` doesn't read anything from filesystem
    (content) => dockerfilelint.run(configObject, content)
);
```

**Comment by EvgenyOrekhov** [comment]
I don't care anymore.
